cetmix_tower_server/tests/common.py

```diff
@@ -15,6 +15,10 @@ def setUp(self, *args, **kwargs):
         # Cetmix Tower helper model
         self.CetmixTower = self.env["cetmix.tower"]
 
+        # Tags
```

Comment:
You could create a new Tag model similar to how it's done for the [Server](https://github.com/cetmix/cetmix-tower/blob/a44e50b7b0f0614d2f89ecff514f60246ad9ae74/cetmix_tower_server/tests/common.py#L36). 
And use it later for creating new tags more convenient way.